commons/c2cgeoportal_commons/models/static.py

```diff
@@ -103,6 +103,7 @@ class User(Base):
                                  info={'colanderalchemy': {'exclude': True}})
     role_name = Column(String, info={
         'colanderalchemy': {
+            'title': _('role'),
```

Comment:
Note that I put uppercase on first character in other titles

Reply:
This is true for wms layer... But false for role and user. So, lower or uppercase ?